app/utils.py

```diff
@@ -7,22 +7,31 @@
 
 
 # ------------------------------------------------------------------------------
-# Content
+# Content  : Those functions aim at returning either the name of :
+#  the hovered department
+#  the hovered camera (marker)
+# ------------------------------------------------------------------------------
 
 map_style = {'width': '100%',
              'height': '75vh',
              'margin': 'auto',
              'display': 'block'}
 
 
-def get_info(feature=None):
-
-    header = [html.H4('Département sélectionné :')]
+def get_info(feature=None, feature_type=None):
 
-    if not feature:
-        return header + [html.P('Faites glisser votre curseur sur un département')]
+    '''This function check the geojson object thanks to the feature_type argument
+    and a return the appropriate message.'''
 
-    return header + [html.B(feature['properties']['nom'])]
+    header_dept = [html.H4('Département sélectionné :')]
+    header_camera = [html.H4('Caméra sélectionnée :')]
+    if feature:
+        if feature_type == 'markers':
+            return header_camera + [html.B('Zone: {}'.format(feature['properties']['area']))]
+        elif feature_type == 'geojson_alerts' or feature_type == 'geojson_risks':
```

Comment:
It probably does not lead to any substantial gain in terms of performance but writing

```elif feature_type in ['geojson_alerts', 'geojson_risks']:```

could work with only one condition to verify, maybe?

Reply:
thanks @pechouc  for your feedback. 
> I just had a question about the `marker_click` callback you have added in `main.py`. Is it actually printing the formatted text when the user clicks on the marker? I thought you said during the last meeting that we could only display the name of the device in the upper-right info box, but I might have missed the latest developments of the topic!

No it's not. I left the beginning of this function to complete it as soon as I'd have the answer but it's not displaying info on a popup for now.

Reply:
> It probably does not lead to any substantial gain in terms of performance but writing
> 
> `elif feature_type in ['geojson_alerts', 'geojson_risks']:`
> 
> could work with only one condition to verify, maybe?

You're right. It's probably not a big change in term of speed but definitely cleaner. Will change it